src/Chapter2.hs

```diff
@@ -740,7 +760,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains a [] = []
+contains a (x:xs)
+  | elem a x = x : contains a xs
+  | otherwise = contains a xs
```

Comment:
Here you also wrote the function, that is similar to the standard `filter` function 👍🏼